src/components/toast/toast.scss

```diff
@@ -0,0 +1,128 @@
+@import "mixins";
+
+@mixin inherit-color() {
+  color: inherit;
+  background-color: inherit;
+}
+
+.show {
+  opacity: 1 !important;
+  animation:toastAnimationFadeIn 1s;
+
+  &:hover {
+    animation-play-state: paused;
+  }
+}
+
+.hide {
+  transition: all 1s;
+  animation: toastAnimationFadeOut 0.5s;
+}
+
+.toast {
+  ::before {
+    display: block;
+  }
+
+  position: relative;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  border-radius: 8px;
```

Comment:
Don't we have some border-radius class on ds?

Reply:
no, we don't have, and I don't know if it would be worth it, because we need to follow the design requirements